src/Analytics/Analytics.model.ts

```diff
@@ -0,0 +1,82 @@
+import { QueryResult } from 'pg'
+import { SQL } from 'decentraland-server'
+import { analytics } from '../database'
+import { WeeklyStats } from './Analytics.types'
+
+export const CONNECTION_ERROR = 'connection terminated'
+export const TIMEOUT_ERROR = 'ETIMEDOUT'
+const TIMEOUT_MS = 30000
+
+export class Analytics {
+  static db = analytics
+
+  static async getWeekly(
+    base: string,
+    retry = true
+  ): Promise<WeeklyStats | null> {
+    try {
+      const result = await Promise.race([
+        Analytics.db.client.query(
+          SQL`SELECT * 
+            FROM analytics.scene_open_data_weekly 
+            WHERE base = ${base} 
+            ORDER BY week DESC 
+            LIMIT 1`
+        ),
+        // force a timeout at 30 seconds, because sometimes the connection hangs indefinetely
+        new Promise<QueryResult<any>>((_, reject) =>
+          setTimeout(() => reject(new Error(TIMEOUT_ERROR)), TIMEOUT_MS)
+        )
+      ])
+
+      let weekly: WeeklyStats | null = null
+
+      if (result.rows.length > 0) {
+        const {
+          week,
+          title,
+          base,
+          users,
+          sessions,
+          median_session_time,
+          min_session_time,
+          average_session_time,
+          max_session_time,
+          direct_users,
+          direct_sessions,
+          max_concurrent_users,
+          max_concurrent_users_time
+        } = result.rows[0]
+
+        weekly = {
+          week,
+          title,
+          base,
+          users: parseInt(users, 10),
```

Comment:
Are we parsing it for some special reason?

Reply:
becuase they are returned by the query as strings instead of numbers

Reply:
Yeah! I'm wondering if we can display as they are in the front-end as we don't need to manipulate them